### ISSUE TITLE
Problem trying to whitelist rgb color within style attribute

### ISSUE THREAD

**Issue by tquill** [rails/rails-html-sanitizer#68]
I have a basic comment scrubber:

    class CommentScrubber < Rails::Html::PermitScrubber
      def initialize
        super
        self.tags = %w( strong em u div span br h1 h2 h3 h4 ul ol li table thead tbody th tr td img hr a )
        self.attributes = %w( style colspan rowspan text-align class href target src href )
      end

      def skip_node?(node)
        node.text?
      end
    end

The problem is if a comment is styled with a color using an rgb value, the color gets stripped out.

For example, this doesn't work:

    <span style="color: rgb(226, 80, 65);">PRODUCTS</span>

But this does work:

    <span style="color: red;">PRODUCTS</span>

Any suggestions on how to get the rgb color working properly?

**Comment by fschwahn** [comment]
I couldn't reproduce this:
```ruby
scrubber = CommentScrubber.new
html_fragment = Loofah.fragment('<span style="color: rgb(226, 80, 65);">PRODUCTS</span>')
html_fragment.scrub!(scrubber)
html_fragment.to_s
# => "<span style=\"color: rgb(226, 80, 65);\">PRODUCTS</span>"
```

Seems like this was fixed in loofah 2.2.0 (see https://github.com/flavorjones/loofah/blob/master/CHANGELOG.md#220--2018-02-11), and is not an issue with rails-html-sanitizer.

**Comment by flavorjones** [comment]
Closing, @fschwahn is correct, this was fixed upstream in Loofah.
